docs/OV_Runtime_UG/supported_plugins/Device_Plugins.md

```diff
@@ -38,17 +40,17 @@ Devices similar to the ones we have used for benchmarking can be accessed using
 ## Features support matrix
 The table below demonstrates support of key features by OpenVINO device plugins.
 
-| Capability | CPU | [GPU](./GPU.md) | GNA | VPU |
-| ---------- | --- | --- | --- | --- |
-| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? |
-| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? |
-| [Automatic batching](../automatic_batching.md) | No | Yes | ? | ? |
-| [Multi-stream execution](@ref openvino_docs_optimization_guide_dldt_optimization_guide) | Yes | Yes | ? | ? |
-| [Models caching](../Model_caching_overview.md) | Yes | Partial | ? | ? |
-| [Dynamic shapes](../ov_dynamic_shapes.md) | Yes | Partial | ? | ? |
-| Import/Export | Yes | No | ? | ? |
-| [Preprocessing acceleration](../preprocessing_overview.md) | Yes | Yes | ? | ? |
-| [Stateful models](../network_state_intro.md) | Yes | No | ? | ? |
-| [Extensibility](@ref openvino_docs_Extensibility_UG_Intro) | Yes | Yes | ? | ? |
+| Capability | CPU | [GPU](./GPU.md) | GNA | VPU | [ARM CPU](ARM_CPU.md) |
+| ---------- | --- | --- | --- | --- | --- |
+| [Heterogeneous execution](../hetero_execution.md)| Yes | Yes | ? | ? | No |
+| [Multi-device execution](../multi_device.md) | Yes | Yes | ? | ? | No |
```

Comment:
As far as I know multi-device execution spread independent simultaneously executed infer requests between different devices so should be supported by Arm CPU plugin

Reply:
Yes we support work under multi and hetero plugin